.gitignore

```diff
@@ -1,7 +1,8 @@
 .DS_Store
 __pycache__/
 *.swp
-.env
+pegasus.env
```

Comment:
Since these don't contain any secrets, would it be ok to check them in? (With a comment inside reminding people not to put any secrets inside!) If we don't add these, should probably at least have an example one for the default case. The concern is that we want it to be as clear as possible to someone bringing up their own server what they need to add, variables they need to set, etc.